scapy/volatile.py

```diff
@@ -648,6 +648,13 @@ def _fix(self):
         return time.strftime("%y%m%d%H%M%SZ",time.gmtime(time.time()+self.diff))
 
 
+class GeneralizedTime(AutoTime):
+    def __init__(self, diff=None):
```

Comment:
Wouldn't it be better to have `diff=0` here, since `._fix()` will raise a `TypeError` exception when `diff` is left to its default value?

Also, please consider complying to PEP-8 coding style (space around assignment `=`, limited line length):

```python
class GeneralizedTime(AutoTime):
    def __init__(self, diff=0):
        self.diff = diff
    def _fix(self):
        return time.strftime("%Y%m%d%H%M%SZ",
                             time.gmtime(time.time()+self.diff))